agir/groups/components/groupPage/GroupSettings/GroupManagementPage/MainPanel.js

```diff
@@ -144,6 +148,47 @@ const MainPanel = (props) => {
       )}
     </>
   );
+
+  const managerMainPanel = (
+    <>
+      <StyledTitle>Gestion et animation</StyledTitle>
+      <span>
+        Vous êtes gestionnaires du groupe{" "}
```

Comment:
Pour l'instant on utilise deux mots différents pour les **groupes** de la France insoumise et les **équipes** de soutien pour la campagne présidentielle. C'est pour ça qu'on a créé un tag de template de chaînes de caractères `withGroupWords` pour choisir automatiquement le bon mot selon les données du groupe. Tu peux l'utiliser comme ça : 
```suggestion
        {withGroupWord`Vous êtes gestionnaires du groupe `}
```